ticdc: fix sink uri for storage sink (#13445)
pingcap/docs: ticdc/ticdc-sink-to-cloud-storage.md

> 

Author: `ticdc/ticdc-changefeed-config.md` in https://github.com/pingcap/docs-cn/pull/13654/files has been translated in a separate PR: https://github.com/pingcap/docs/pull/13357

> ## Configure sink URI

Author: `ticdc/ticdc-changefeed-config.md` in https://github.com/pingcap/docs-cn/pull/13654/files has been translated in a separate PR: https://github.com/pingcap/docs/pull/13357

> This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems and its format is as follows:

Author: ```suggestion
This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems. The format of Sink URI is as follows:
```

> This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. The Sink URI is used to specify the connection information for TiCDC downstream systems. The format of Sink URI is as follows:

Reviewer: ```suggestion
This section describes how to configure Sink URI for storage services, including Amazon S3, GCS, Azure Blob Storage, and NFS. Sink URI is used to specify the connection information of the TiCDC target system. The format is as follows:
```

ref: https://docs.pingcap.com/tidb/dev/ticdc-sink-to-kafka

```shell
[scheme]://[host]/[path]?[query_parameters]
```

For `[query_parameters]` in the URI, the following parameters can be configured:

| Parameter | Description | Default value | Value range |
| :---------| :---------- | :------------ | :---------- |
| `worker-count` | Concurrency for saving data changes to cloud storage in the downstream.  | `16` | `[1, 512]` |
| `flush-interval` | Interval for saving data changes to cloud storage in the downstream.   | `5s` | `[2s, 10m]` |
| `file-size` | A data change file is stored to cloud storage if the number of bytes exceeds the value of this parameter. | `67108864` | `[1048576, 536870912]` |
| `protocol` | The protocol format of the messages sent to the downstream.  | N/A |  `canal-json` and `csv` |

> | `enable-tidb-extension` | When `protocol` is set to `canal-json` and `enable-tidb-extension` is set to `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to canal-json messages. | `false` | `false` and `true` |

Reviewer 2: ```suggestion
| `enable-tidb-extension` | When `protocol` is set to `canal-json` and `enable-tidb-extension` is set to `true`, TiCDC sends [WATERMARK events](/ticdc/ticdc-canal-json.md#watermark-event) and adds the [TiDB extension field](/ticdc/ticdc-canal-json.md#tidb-extension-field) to Canal-JSON messages. | `false` | `false` and `true` |
```

> **Note:**
>
> Data change files are saved to the downstream when either `flush-interval` or `file-size` meets the requirements.
> The `protocol` parameter is mandatory. If TiCDC does not receive this parameter when creating a changefeed, the `CDC:ErrSinkUnknownProtocol` error is returned.

### Configure sink URI for external storage

The following is an example configuration for Amazon S3:

```shell
--sink-uri="s3://bucket/prefix?protocol=canal-json"
```

The following is an example configuration for GCS:

```shell
--sink-uri="gcs://bucket/prefix?protocol=canal-json"
```

The following is an example configuration for Azure Blob Storage:

```shell
--sink-uri="azure://bucket/prefix?protocol=canal-json"
```

> **Tip:**
>
> The URI parameters of Amazon S3, GCS, and Azure Blob Storage in TiCDC are the same as their URI parameters in BR. For details, see [Backup storage URI format](/br/backup-and-restore-storages.md#uri-format-description).

### Configure sink URI for NFS

The following is an example configuration for NFS:

```shell

> --sink-uri="file:///my-directory/prefix?protocol=canal-json"

Reviewer 2: /// 三个斜杠吗？

> --sink-uri="file:///my-directory/prefix?protocol=canal-json"

Reviewer 2: 应该是2个斜杠吧 
```suggestion
--sink-uri="file://my-directory/prefix?protocol=canal-json"
```

Reviewer: 前两个是协议分隔符，第三个是 linux 根目录
